### DISCUSSION TITLE
Welcome to Learncpp_CN Discussions!

### DISCUSSION THREAD

**Discussion by hanxiaomax** [hanxiaomax/Learncpp_CN#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cugerQDHJ** [comment]
大佬辛苦，给大佬递茶

**Reply by hanxiaomax** [reply]
赠人玫瑰手留余香。感谢你的支持

**Comment by PanBharal** [comment]
感谢大佬！终于找到有人翻译中文版了，之前一直用edge网页翻译，有些词也不够准确，而且原作者网站广告是真的讨厌！支持大佬！会给周边同学推荐的！

**Comment by mondlichthari** [comment]
good
